gnss_to_map_convertor/include/gnss_to_map_convertor/GNSSToMapNode.h

```diff
@@ -0,0 +1,89 @@
+#pragma once
+
+/*
+ * Copyright (C) 2018-2019 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#include <ros/ros.h>
+#include <tf2_ros/transform_listener.h>
+#include <cav_srvs/GetTransform.h>
+#include <wgs84_utils/wgs84_utils.h>
+#include <message_filters/subscriber.h>
+#include <message_filters/time_synchronizer.h>
+#include <message_filters/sync_policies/approximate_time.h>
+#include <sensor_msgs/NavSatFix.h>
+#include <novatel_gps_msgs/NovatelDualAntennaHeading.h>
+#include <novatel_gps_msgs/NovatelPosition.h>
+#include <novatel_gps_msgs/NovatelXYZ.h>
+#include <gnss_to_map_convertor/GNSSToMapConvertor.h>
+#include <carma_utils/CARMAUtils.h>
+
+/**
+ * \class GNSSToMapNode
+ * \brief ROS Node which maintains a tf2 transform tree which can be accessed by other nodes.
+ *
+ * The get_transform service can be used to obtain coordinate transformations between two frames.
+ * Only transforms published on the /tf or /tf_static topics are recorded by this node.
+ */
+class GNSSToMapNode {
```

Comment:
Do we need to put this class in its own namespace?

Reply:
I don't think so. This class would never be included in a library for usage outside this package.